homeassistant/components/intelliclima/entity.py

```diff
@@ -0,0 +1,37 @@
+"""Platform for shared base classes for sensors."""
+
+from pyintelliclima import IntelliClimaC800, IntelliClimaECO
+
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import IntelliClimaCoordinator
+
+
+class IntelliClimaEntity(CoordinatorEntity[IntelliClimaCoordinator]):
+    """Define a generic class for IntelliClima entities."""
+
+    _attr_attribution = "Data provided by unpublished IntelliClima API"
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: IntelliClimaCoordinator,
+        device: IntelliClimaECO | IntelliClimaC800,
+        description: EntityDescription,
+    ) -> None:
+        """Class initializer."""
+        super().__init__(coordinator=coordinator)
+
+        self.entity_description = description
+        self._attr_unique_id = f"{device.id}_{description.key}"
+
+        # Make this HA "device" use the IntelliClima device name.
+        self._attr_device_info = {
+            "identifiers": {(DOMAIN, device.id)},
+            "name": device.name,
```

Comment:
I say that this integration is marked as `cloud_polling`. Can you add links to cloud panel as configuration URL?

Reply:
Unfortunately no open (web) API exists, so adding a configuration_url is not possible and will be left out. The API is implemented by me by reverse engineering in [pyintelliclima](https://github.com/dvdinth/pyintelliclima)

Reply:
So these devices can only be managed via the app? Is there no web portal?